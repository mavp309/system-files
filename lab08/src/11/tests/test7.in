Ravi
29
99001122
80
