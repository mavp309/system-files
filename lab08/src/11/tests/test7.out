Name: Ravi
Age: 29
Roll: 99101122
Marks: 80
